.github/workflows/stale.yml

```
@@ -29,10 +29,9 @@ jobs:
           stale-pr-label: "stale"       # label to use when marking as stale
 
           stale-pr-message: >
-            This PR had no visible activity in the past 60 days, labeling it as stale.
-            Any new activity will remove the stale label. To attract more reviewers, please tag
-            someone or notify the dev@solr.apache.org mailing list.
+            This PR has had no activity for 60 days and is now labeled as stale. 
+            Any new activity or setting it as a draft will remove the stale label. 
```

I like the mention of 'draft' as a choice. How about rewording to make it closer to the 'Convert to draft' UI element?
```suggestion
            Any new activity or converting it to draft will remove the stale label. 
```